{"name": "Попытка задания email без @", "status": "skipped", "statusDetails": {"message": "XFAIL API принимает невалидный email (без символа @). Ожидается ошибка валидации (400),         но получаем 200.\n\nAssertionError: Ожидался статус 400, но получен 200. Тело ответа: {\"success\":true,\"user\":{\"email\":\"invalid-email-e6eee602-d84e-40da-8743-793ebdc76657@test\",\"name\":\"User_c7614e91\"}}\nassert 200 == 400\n +  where 200 = <Response [200]>.status_code", "trace": "self = <tests.test_user_profile.TestUserProfile object at 0x000002C4185160F0>, api_client = <helpers.api_client.StellarBurgersApi object at 0x000002C41865AB70>\nregistered_user = {'access_token': 'Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IjY4NDg1NDllOWVkMjgwMDAxYjYxNjg4YSIsImlhdCI6MTc0...2Mo2lODcqaSIEaZpT7M', 'email': 'user_225f93d1@test.com', 'name': 'User_c7614e91', 'password': 'Password_2a70ba86', ...}\n\n    @allure.story(\"Ошибки валидации\")\n    @allure.title(\"Попытка задания email без @\")\n    @pytest.mark.xfail(\n        reason=\"API принимает невалидный email (без символа @). Ожидается ошибка валидации (400), \\\n        но получаем 200.\")\n    def test_invalid_email_no_at(self, api_client, registered_user):\n        invalid_email = f\"invalid-email-{uuid.uuid4()}@test\"\n        response = api_client.update_user_info(\n            registered_user[\"access_token\"],\n            invalid_email,\n            registered_user[\"name\"]\n        )\n>       assert response.status_code == HTTP_STATUS[\"BAD_REQUEST\"], (\n            f\"Ожидался статус 400, но получен {response.status_code}. \"\n            f\"Тело ответа: {response.text}\"\n        )\nE       AssertionError: Ожидался статус 400, но получен 200. Тело ответа: {\"success\":true,\"user\":{\"email\":\"invalid-email-e6eee602-d84e-40da-8743-793ebdc76657@test\",\"name\":\"User_c7614e91\"}}\nE       assert 200 == 400\nE        +  where 200 = <Response [200]>.status_code\n\ntests\\test_user_profile.py:98: AssertionError"}, "attachments": [{"name": "log", "source": "59dd6d9b-657d-4f6c-a522-5099b5461b72-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "5241bf69-574d-41fd-8377-95bc3706146c-attachment.txt", "type": "text/plain"}], "start": 1749570725140, "stop": 1749570725422, "uuid": "08b615d7-32dd-44b6-a0d8-93a2b990d0d3", "historyId": "83085b3c54241accc61f9fd746fa95d8", "testCaseId": "83085b3c54241accc61f9fd746fa95d8", "fullName": "tests.test_user_profile.TestUserProfile#test_invalid_email_no_at", "labels": [{"name": "story", "value": "Ошибки валидации"}, {"name": "feature", "value": "Изменение данных пользователя"}, {"name": "epic", "value": "Stellar Burgers API"}, {"name": "tag", "value": "@pytest.mark.xfail(reason='API принимает невалидный email (без символа @). Ожидается ошибка валидации (400),         но получаем 200.')"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_user_profile"}, {"name": "subSuite", "value": "TestUserProfile"}, {"name": "host", "value": "Gemini_78"}, {"name": "thread", "value": "7332-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_user_profile"}]}